wire/wire.proto

```diff
@@ -0,0 +1,13 @@
+syntax = "proto3";
+package basictracer_go.wire;
```

Comment:
Certainly true... `basictracer` is already making assumptions about which tracer state is worth keeping around, though. (FWIW, it wouldn't quite work for dapper, either: there were a few other (literal) bits sent over the wire, not just a `sampled` bool)